python/GangaLHCb/Lib/Applications/BenderBox.py

```diff
@@ -0,0 +1,520 @@
+# -*- coding: utf-8 -*-
```

Comment:
These are not necessary unless the code contains non-ASCII (unicode) characters. This I suspect was put in by default by the IDE